config/deploy.production.yml

```diff
@@ -0,0 +1,28 @@
+servers:
+  web:
+    hosts:
+      - 44.196.164.25
+    labels:
+      traefik.http.routers.micarrera-web-production.rule: Host(`micarrera.uy`)
```

Comment:
Should we also add `fing.micarrera.uy` too?

Reply:
added in [803422b](https://github.com/cedarcode/mi_carrera/pull/349/commits/803422bde48dfcf1b96b8cb0c76fd920dc9ecbef)